assignments/functions.md

```diff
@@ -68,34 +97,67 @@ Is there any difference in the behavior of these two variants? If there is what
 
 ```js
 // Your code goes here
+let x = +prompt("enter an integer");
```

Comment:
The function says to write a function that takes two parameters  returns a value. It never said you need to take the value from user using `prompt`. This portion is all that needed
```
function pow(x,n) {
  if (n <= 1) {
  return ("integers less then 1 is not allowed");
  }
  return x ** n ;
}
```